app/templates/components/curriculum-inventory-sequence-block-session-list.hbs

```diff
@@ -38,7 +38,12 @@
             <td class='text-center' colspan=2>
               {{if (is-in linkedSessions session) (t 'general.yes') (t 'general.no')}}
             </td>
-            <td class='text-left' colspan=3>{{session.title}}</td>
+            <td class='text-left' colspan=3>
+              {{session.title}}
+              {{#if session.isIndependentLearning}}
+                <strong>({{t 'general.ilm'}})</strong>
```

Comment:
After a quick visual review, I think we want to prepend, rather than append the `(ILM)` label to the session title. This will make it visually a little easier to parse.

Reply:
on it.